libs/platform/src/lib/table/table.component.ts

```diff
@@ -1979,6 +1992,52 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
     }
 
     /** @hidden */
+    private _listenToFocus(): void {
+        if (!this.tableContainer || !this.focusableMock) {
+            return;
+        }
+
+        let shouldDelegateFocus = true;
+
+        this._subscriptions.add(
+            fromEvent(this.tableContainer.nativeElement, 'keydown').subscribe((event: KeyboardEvent) => {
+                if (this.loading) {
+                    event.preventDefault();
+                    return;
+                }
+
+                if (KeyUtil.isKeyCode(event, TAB)) {
+                    this._tableService.processFocusBetweenCellInners(
+                        event,
+                        this.tableContainer.nativeElement,
+                        this.focusableMock.nativeElement
+                    );
+                }
+            })
+        );
+
+        this._subscriptions.add(
+            fromEvent(this.tableContainer.nativeElement, 'focusout')
+                .pipe(delay(0))
+                .subscribe(() => {
+                    if (!this.tableContainer.nativeElement.contains(this._document.activeElement)) {
+                        this._tableService.resetFocusedTableCell();
+                        this._cdr.markForCheck();
+                    }
+                })
+        );
+
+        this._subscriptions.add(
+            fromEvent(this.focusableMock.nativeElement, 'focus').subscribe(() => {
+                if (shouldDelegateFocus) {
+                    this._tableService.focusNextTableCell(FIRST_CELL_NAVIGATION_ID);
+                }
+
+                shouldDelegateFocus = true;
```

Comment:
as far as I can see `shouldDelegateFocus` is always true, don't understand why we need it

Reply:
removed